app/forms/idv/api_image_upload_form.rb

```diff
@@ -304,6 +313,10 @@ def throttle
       )
     end
 
+    def throttled?
+      @throttled = throttle.throttled? if document_capture_session
```

Comment:
It looks like you stopped using `@throttled` elsewhere, so you don't need to set it here.
```suggestion
      throttle.throttled? if document_capture_session
```